src/main/java/no/sikt/nva/pubchannels/dataporten/DataportenJournal.java

```diff
@@ -0,0 +1,110 @@
+package no.sikt.nva.pubchannels.dataporten;
+
+import static java.util.Objects.isNull;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import no.sikt.nva.pubchannels.Immutable;
+import no.sikt.nva.pubchannels.handler.ScientificValue;
+import no.sikt.nva.pubchannels.handler.ThirdPartyJournal;
+
+/* default */ final class DataportenJournal implements Immutable, ThirdPartyJournal {
+    private static final String YEAR_FIELD = "Year";
+    private static final String ONLINE_ISSN_FIELD = "Eissn";
+    private static final String IDENTIFIER_FIELD = "Pid";
+    private static final String SCIENTIFIC_VALUE_FIELD = "Level";
+    private static final String PRINT_ISSN_FIELD = "Pissn";
```

Comment:
should pissn and eissn now be named onlineIssn and printedIssn?

Reply:
Field names has to match openApi right?

Reply:
Or is this against the database?

Reply:
This maps the response from the channel registry. As the class implements ExternalJournal, we use field names that makes sense in our scope. So this is correct.